core/src/main/java/org/kohsuke/stapler/export/Model.java

```diff
@@ -190,6 +190,9 @@ void writeNestedObjectTo(T object, TreePruner pruner, DataWriter writer, Set<? e
                 superBlacklist.add(p.name);
             }
             superModel.writeNestedObjectTo(object, pruner, writer, superBlacklist);
+        } else {
+            writer.name("class");
```

Comment:
I would suggest `$class` for consistency with the meta-attribute used in form binding.